src/test/java/api/support/APITestContext.java

```diff
@@ -31,6 +31,7 @@
   private static final String USER_ID = "79ff2a8b-d9c3-5b39-ad4a-0a84025ab085";
 
   private static final String TOKEN = "eyJhbGciOiJIUzUxMiJ9eyJzdWIiOiJhZG1pbiIsInVzZXJfaWQiOiI3OWZmMmE4Yi1kOWMzLTViMzktYWQ0YS0wYTg0MDI1YWIwODUiLCJ0ZW5hbnQiOiJ0ZXN0X3RlbmFudCJ9BShwfHcNClt5ZXJ8ImQTMQtAM1sQEnhsfWNmXGsYVDpuaDN3RVQ9";
+  private static final String PATRON_GROUP_ID = "3a60c6ac-a93a-11e9-a2a3-2a2ae2dbcce4";
```

Comment:
Why does this need to be global state for all of the tests? 

Could the tests that care about patron groups set up their own patron groups instead?

Reply:
So, you want to remove this constant from APITestContext, and replace this value to each test regarding patron group mechanism?

Reply:
Yes, I think it easier to understand a test the more context is within the test (at an appropriate abstraction).

There is already too much shared state in the API tests, I’d prefer not to add more.

Not using fixed IDs also leads to less accidental coupling between the test and the environment (e.g. the fakes cannot assume which record is being asked for)

Reply:
Ok, I done it